src/Forms/ChangeCasing.cs

```diff
@@ -95,7 +95,7 @@ internal void FixCasing(Subtitle subtitle, string language)
                                 prev = p.Text[match.Index - 1].ToString(CultureInfo.InvariantCulture);
                             if (match.Index + 1 < p.Text.Length)
                                 next = p.Text[match.Index + 1].ToString(CultureInfo.InvariantCulture);
-                            if (prev != ">" && next != ">")
+                            if (prev != "<" || next != ">")

```

Comment:
Well, seems a bit strange... I've tested somewhat and that regular expression is not the best for cases like "but<i>i was<i> bad" - I've tried to fix it here: c2a3d33f23a9f977e5283951e25399adb2eab3c9